dgraph/cmd/zero/assign.go

```diff
@@ -196,7 +196,7 @@ func (s *Server) AssignUids(ctx context.Context, num *pb.Num) (*pb.AssignedIds,
 
 	select {
 	case <-ctx.Done():
-		return reply, ctx.Err()
+		return &emptyAssignedIds, ctx.Err()
```

Comment:
why do you need to change this?

Reply:
There is a race condition with the use of reply. This happens when ctx.Done was called but lease is still working to assign uids